install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd build && cmake -G "Unix Makefiles" .. && make || {
+    echo
+    echo "Doxygen build failed."
+    echo
+}
```

Comment:
In my opinion this is script should not be in the repository, it is a private script used by the user.

Reply:
I agree

Reply:
Is this good to go: on removal of the above script?

Reply:
I don't think so see my comment about the old style comment.